### PR TITLE
Bug Fix Add Cluster Description to card and Big Spinner

### PR DESCRIPTION
**Pull request by Mubangizi**

### CODE REVIEW

**Comment by Mubangizi** [comment]
> @Mubangizi good stuff, kindly not that branch name format is alien, ought it not to be `bg-fix-cluster-desc` and "fix(Cluster Desc) adds Cluster Description to card and Big Spinner" to be `Bug Fix Add Cluster Description to card and Big Spinner`

Thanks for pointing that out. its been noted

File: src/components/ClustersList/index.jsx [hunk 1]
```diff
@@ -16,34 +17,58 @@ class ClustersList extends Component {
...
 
     return (
       <div className="ClusterList">
-        {isRetrieved ? (clusters.clusters.map((cluster) => (
-          <Link to={{ pathname: `/clusters/${cluster.id}/resources` }} key={cluster.id}>
-            <div key={cluster.id} className="ClusterCardItem">
-              <ClusterCard name={cluster.name} description={cluster.host} icon={crane} />
+        {
+          isRetrieving ? (
+            <tr className="TableLoading">
```
Comment (neelxie) [inline src/components/ClustersList/index.jsx]: @Mubangizi kindly help me understand why we have table row tags minus the actual table in this file?

File: src/components/ClustersList/index.jsx [hunk 2]
```diff
@@ -16,34 +17,58 @@ class ClustersList extends Component {
...
+                    <ClusterCard
+                      name={cluster.name}
+                      description={cluster.description}
+                      icon={crane}
+                    />
+                  </div>
+                </Link>
+              )))
+                : (
+                  <tr>
```
Comment (neelxie) [inline src/components/ClustersList/index.jsx]: And here, I would suggest maybe have a representative div instead.

**Comment by Mubangizi** [comment]
Resolved @neelxie  thanks

**Pull request merged**